timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -774,6 +774,13 @@ module.exports = [
           },
         ],
       },
+      {
+        title: "SkipScan",
+        href: "skipscan",
+        tags: ["skipscan", "query", "timescaledb"],
+        keywords: ["SkipScan", "query", "timescaledb"],
+        excerpt: "Use SkipScan to speed up your TimescaleDB queries",
+      },
```

Comment:
This should stay under "query data" in some way. And, I think it's best to refer to it as DISTINCT queries, not SkipScan. Most people in the world don't know what that is (even in Postgres). This get to some of the conversation we've had around a better "query best practices/tips" section that has yet to materialize.

The excerpt, at least, would be "faster DISTINCT queries with time-series data and TimescaleDB". something like that.